Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -115,6 +115,14 @@ public final class ManifestLoader {
         let toml = try TOMLItem.parse(tomlString)
         let package = PackageDescription.Package.fromTOML(toml, baseURL: baseURL)
         let products = PackageDescription.Product.fromTOML(toml)
+        let errors = parseErrors(toml)
+
+        guard errors.isEmpty else {
+            // FIXME: Output with proper diagnostics infrastucture when available.
+            print("Found errors while parsing manifest of package \(package.name):")
+            errors.forEach { print($0) }
```

Comment:
Something like:
```
error: invalid manifest format; invalid version string: 1.00
```
is in keeping with the Swift diagnostic style, which I would like us to follow.